orttraining/orttraining/models/runner/training_runner.h

```diff
@@ -149,6 +149,8 @@ class TrainingRunner {
 
     int data_parallel_size = 1;
     int horizontal_parallel_size = 1;
+    // Enable gradient clipping.
+    bool enable_gradient_clip=true;
```

Comment:
>true [](start = 30, length = 4)

What's our current default behavior for gradient clipping? 

Is it on ?

Reply:
it's got different hard-coded behaviors in different optimizer builders. For Adam. default is off, for lamb is on..  It's better to have a control knob for this.

Reply:
Right... So our Adam's default behavior would be different with this change 

Please let the team know... 

---
In reply to: [395801750](https://github.com/microsoft/onnxruntime/pull/3243#discussion_r395801750) [](ancestors = 395801750)